src/modules/fetchData.js

```diff
@@ -0,0 +1,38 @@
+async function sendAPI(url, data = {}) {
```

Comment:
- [ ] Please make sure to apply the changes required by the previous reviewer. You should use arrow functions instead of creating functions using the `function` keyword. Just as a reminder, arrow functions look something like this:
`const namOfMyFunction = () => { code that my function should execute }`